projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -812,7 +791,7 @@ function get_locked_content_placeholder_text( $newsletter_access_level ) {
 	// Only display this text when Stripe is connected and the post is marked for paid subscribers
 	if (
 		$newsletter_access_level === 'paid_subscribers'
-		&& ! empty( \Jetpack_Memberships::get_connected_account_id() )
+		&& ! empty( Jetpack_Memberships::get_connected_account_id() )
```

Comment:
Is this change needed?  (I saw @millerf 's comment about the other `\` being needed...)

Reply:
I could be wrong here but I think his suggestion was related to `class_exists`. For `Jetpack_Memberships` here I added `use Jetpack_Memberships` to the top of this file.

Reply:
Yup, sounds good.